.env

```diff
@@ -0,0 +1 @@
+export SOUNDCLOUD_CLIENT_ID=69e93cf2209402f6f3137a6452cf498f
```

Comment:
Yeah, `app/assets/javascripts/plays.js` should be updated too.
Don't know how to get the acceptance tests to pass without it though ...